### PR TITLE
Router: "raw" prefix is removed for low-level wrappers

### PR DESCRIPTION
**Pull request by sgrishchenko**
@turansky @aerialist7

### CODE REVIEW

File: kotlin-react-router-dom/src/main/kotlin/react/router/dom/useRouteMatch.kt [hunk 1]
```diff
@@ -3,7 +3,6 @@
 
 package react.router.dom
 
-@JsName("useRouteMatch")
-external fun rawUseRouteMatch(
+external fun useRouteMatch(
     options: MatchOptions?,
```
Comment (turansky) [inline kotlin-react-router-dom/src/main/kotlin/react/router/dom/useRouteMatch.kt:7]: definedExternally?

Comment (sgrishchenko) [reply kotlin-react-router-dom/src/main/kotlin/react/router/dom/useRouteMatch.kt]: yea, thanks, fixed

File: kotlin-react-router-dom/src/main/kotlin/react/router/dom/useRouteMatch.ext.kt [hunk 2]
```diff
@@ -9,7 +9,7 @@ fun useRouteMatch(
     sensitive: Boolean = false,
 ): Match? {
     if (path.isEmpty()) {
```
Comment (turansky) [inline kotlin-react-router-dom/src/main/kotlin/react/router/dom/useRouteMatch.ext.kt]: Never called :)

**Review by turansky — CHANGES_REQUESTED** [review]

File: kotlin-react-router-dom/src/main/kotlin/react/router/dom/useRouteMatch.ext.kt (hunk 2 above)
Comment (sgrishchenko) [reply kotlin-react-router-dom/src/main/kotlin/react/router/dom/useRouteMatch.ext.kt]: fixed

**Review by turansky — APPROVED** [review]

**Comment by Leonya** [comment]
Released in pre.241

**Pull request merged**